eZ/Bundle/EzPublishCoreBundle/Imagine/PlaceholderProvider/PlacekittenProvider.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Imagine\PlaceholderProvider;
+
+use eZ\Bundle\EzPublishCoreBundle\Imagine\PlaceholderProvider;
+use eZ\Publish\Core\FieldType\Image\Value as ImageValue;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * For demo purposes only: placeholders provided by http://placekitten.com.
+ */
+class PlacekittenProvider implements PlaceholderProvider
```

Comment:
One thought came to me: maybe we could make this more generic, as there are several services that will generate a placeholder based on an URI with height & width. Ex: https://dummyimage.com, https://placeholder.com, https://picsum.photos, https://fakeimg.pl, ... plenty.

My only doubt is the greyscale option. But putting that aside, if you change the constructor to accept an URL as the argument, with placeholder characters, and define the PlaceKitten.com one in the service, it is easy to add support for any of these. And you can rename it to `GeneratorServiceProvider` or something.